include/configs/imx8qm_mek.h

```diff
@@ -286,6 +287,8 @@
 	#define CONFIG_SYS_MMC_ENV_DEV		1  /* USDHC2 */
 	#define CONFIG_MMCROOT			"/dev/mmcblk1p2"  /* USDHC2 */
 #endif
+#endif
```

Comment:
We might also need to define a valid/default CONFIG_MMCROOT (can be emmc), as that is used by base when defining the root parameter.

Reply:
well, in our case we don't use it, as we use custom BOOTCOMMAND
Also it's not defined in Kconfig

Reply:
I think we use it with lmp-base.

```
rsalveti@evatuf:~/build/lmp/layers/meta-lmp$ cat meta-lmp-bsp/recipes-bsp/u-boot/u-boot-base-scr/imx8qm-mek/uEnv.txt.in
bootcmd_dtb=fatload ${devtype} ${devnum}:1 ${fdt_addr} ${fdt_file}
bootcmd_load_k=fatload ${devtype} ${devnum}:1 ${kernel_addr_r} ${boot_file}
bootcmd_run=booti ${kernel_addr_r} - ${fdt_addr}
bootcmd=run set_bootargs; run bootcmd_dtb; run bootcmd_load_k; run bootcmd_run
set_bootargs=setenv bootargs console=ttyLP0,115200 earlycon root=${mmcroot} rootwait rw
```

mmcroot is used there

`        "mmcroot=" CONFIG_MMCROOT " rootwait rw\0" \`

Reply:
fixed

Reply:
Won't this let CONFIG_MMCROOT be undefined as there is no Kconfig for it? I think we can just do an ifdef/else and define CONFIG_MMCROOT.

Reply:
pushed updated version. decided to simplify it completely